feat: add amm migration helper

sakulstra opened the pull request:
## General Changes

This pr adds the address for migration helper on amm pool

## Developer Notes

https://governance.aave.com/t/arfc-deprecate-aave-v2-amm-market/12830/7

---

## Author Checklist

Please ensure you, the author, have gone through this checklist to ensure there is an efficient workflow for the reviewers.

- [x]  The base branch is set to `main`
- [x]  The title is using [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) formatting
- [ ]  The Github issue has been linked to the PR in the Development section
- [x]  The General Changes section has been filled out
- [ ]  Developer Notes have been added (optional)

**If the PR is ready for review:**

- [x]  The PR is in `Open` state and not in `Draft` mode
- [ ]  The `Ready for Dev Review` label has been added

## Reviewer Checklist

Please ensure you, as the reviewer(s), have gone through this checklist to ensure that the code changes are ready to ship safely and to help mitigate any downstream issues that may occur.

- [ ]  End-to-end tests are passing without any errors
- [ ]  Code style generally follows existing patterns
- [ ]  Code changes do not significantly increase the application bundle size
- [ ]  If there are new 3rd-party packages, they do not introduce potential security threats
- [ ]  If there are new environment variables being added, they have been added to the `.env.example` file as well as the pertinant `.github/actions/*` files
- [ ]  There are no CI changes, or they have been approved by the DevOps and Engineering team(s)
- [ ]  Code changes have been quality checked in the ephemeral URL
- [ ]  QA verification has been completed
- [ ]  There are two or more approvals from the core team
- [ ]  Squash and merge has been checked

MareskoY commented:
hello @sakulstra found few problems

1. i try to test migration but for any asset i get error (tested on fresh fork)

```
cannot estimate gas; transaction may fail or may require manual gas limit (error={"reason":"processing response error","code":"SERVER_ERROR","body":"{\"id\":131,\"jsonrpc\":\"2.0\",\"error\":{\"code\":-32000,\"message\":\"execution reverted\",\"data\":{\"error_id\":\"7f88258f-7f9d-4c6b-8928-be671488c306\"}}}","error":{"code":-32000,"data":{"error_id":"7f88258f-7f9d-4c6b-8928-be671488c306"}},"requestBody":"{\"method\":\"eth_estimateGas\",\"params\":[{\"value\":\"0x0\",\"from\":\"0xe4217040c894e8873ee19d675b6d0eec992c2c0d\",\"to\":\"0x3cd1dfb81c50a5300c60a181ed145a7286d81e0a\",\"data\":\"0x3698d492000000000000000000000000000000000000000000000000000000000000008000000000000000000000000000000000000000000000000000000000000000c000000000000000000000000000000000000000000000000000000000000000e0000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000000010000000000000000000000006b175474e89094c44da98b954eedeac495271d0f000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000\"}],\"id\":131,\"jsonrpc\":\"2.0\"}","requestMethod":"POST","url":"https://rpc.tenderly.co/fork/fe8ba4ee-e2f2-49f8-9990-dc8b901a4556"}, method="estimateGas", transaction={"from":"0xE4217040c894e8873EE19d675b6d0EeC992c2c0D","to":"0x3cd1dFB81C50A5300C60a181ED145a7286d81e0a","value":{"type":"BigNumber","hex":"0x00"},"data":"0x3698d492000000000000000000000000000000000000000000000000000000000000008000000000000000000000000000000000000000000000000000000000000000c000000000000000000000000000000000000000000000000000000000000000e0000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000000010000000000000000000000006b175474e89094c44da98b954eedeac495271d0f000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000","accessList":null}, code=UNPREDICTABLE_GAS_LIMIT, version=providers/5.5.3)
```

2. wrong name of v3 market it is not amm v3
<img width="451" alt="Screenshot 2023-06-15 at 14 13 34" src="https://github.com/aave/interface/assets/18032275/451d4e62-f746-42d9-97ed-41700c72da88">

sakulstra commented:
@MareskoY it's 100% same code as before, just added a new address.
All the bugs you might find are already present :)

I think the issue you are facing is related to contract expecting signature with correct chainId, but you are passing 3030 (the code doesn't work well with forks). Vanilla approval should work.

Added a quick patch for the naming.
